src/main/car/functionality/DealershipUI

```diff
@@ -0,0 +1,131 @@
+import javax.swing.*;
+
+public class DealershipUI {
+	static String dealerID = "";
+	
+	private static void makeDealerGUI() {
+	    //dealer information screen
+		JFrame dealerDetails = new JFrame("ICS372 Group 4 Assignment 2");
+		dealerDetails.setLayout(new GridLayout(3,1,0,10));
+		JPanel Options = new JPanel();
+		Options.setLayout(new GridLayout(5,3,10,10));
+		JPanel specificVehicle = new JPanel();
+		specificVehicle.setLayout(new GridLayout());
+		JLabel ID = new JLabel("Dealer: ID here");
+		JLabel Name = new JLabel("Name: ");
+		JLabel Renting = new JLabel("Renting: ");
+		JLabel Acquisition = new JLabel("Vehicle Acquisition: ");
+		JLabel Transfer = new JLabel("Transfer Inventory to: ");
+		JLabel Export = new JLabel("Export: ");
+		JTextField enterName = new JTextField("Dealer Name here");
+		JTextField transferInventory = new JTextField("Receiving Dealer ID here");
+		
+		JButton changeName = new JButton("save");
+		changeName.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton disableRent = new JButton("disable");
+		disableRent.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton enableRent = new JButton("enable");
+		enableRent.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton disableAcquisition = new JButton("disable");
+		disableAcquisition.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton enableAcquisition = new JButton("enable");
+		enableAcquisition.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton transferAllVehicles = new JButton("transfer");
+		transferAllVehicles.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton XMLExport = new JButton("XML");
+		XMLExport.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton JSONExport = new JButton("JSON");
+		JSONExport.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton specificVehicleScreen = new JButton("Specific Vehicle");
+		specificVehicleScreen.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
```

Comment:
here you may want to call Mike's frame creation method or initialize a new Vehicle frame and pass it the selected dealer.

Reply:
I was also thinking about calling Mike's frame creation method and passing in the selected dealer as a parameter if it allows

Reply:
What is the specific Vehicle creation method called? Left the button event handler empty with a comment. Same question with the first screen preceding the dealer search screen

Reply:
Added function based on Mike's pull request. The naming of his elements may change seeing that they are generic. May need additional adjustment to pass the DealerGroup and current working dealer (searchedDealer variable) once Mike's class is initialized.